docs/guide/topics/threads.rst

```diff
@@ -0,0 +1,206 @@
+:orphan:
+
+.. currentmodule:: discord
+
+.. _guide_threads:
+
+Threads
+========
+
+If you've been on a public Discord for a while, you've probably seen a :ddocs:`topics/threads` or two in your time.
+
+They act as offshoots of regular text channels, allowing for a focused, but temporary, location to talk about a given topic.
```

Comment:
```suggestion
They act as offshoots of regular text channels, allowing for a focused, but temporary location to talk about a given topic.
```

Reply:
I am pretty sure the original punctuation is correct.
The 'but temporary' is an off clause, it is not essential to the meaning of the sentence, therefore has to be separated by two commas. You know this, because leaving the off clause out still keeps a valid sentence .
`They act as offshoots of regular text channels, allowing for a focused location to talk about a given topic.`
[See point 3 from this site.](https://owl.purdue.edu/owl/general_writing/punctuation/commas/extended_rules_for_commas.html)